oaknational/Oak-Web-Application
# Fix/lesq 638/eyfs button [LESQ-638]

## Description

Music year: 1978

- Update size of keystage keypad buttons on mobile
- Increase header size on subject listing page on mobile to accomodate buttons
- Change the order of the buttons on mobile 

## How to test

1. Go to https://deploy-preview-2254--oak-web-application.netlify.thenational.academy
2. The keystage selector should look the same

## Screenshots

How it used to look (delete if n/a):
<img width=500  src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/5b0ace18-ccce-4e18-ab95-dc1cec460642">
<img width=500  src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/10ab8d74-6362-47e3-83e2-4f95ba8f9de7">


How it should now look:
<img width=500  src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/4e5b3bf7-bae7-4cf6-9880-c3906">
<img width=500  src="https://github.com/oaknational/Oak-Web-Application/assets/45038878/8ea49c2b-844c-4925-9fe1-cba79ded2063">
c40f6ee)
